py/kubeflow/testing/cleanup_ci.py

```diff
@@ -253,6 +253,44 @@ def cleanup_firewall_rules(args):
   logging.info("Unexpired firewall rules:\n%s", "\n".join(unexpired))
   logging.info("expired firewall rules:\n%s", "\n".join(expired))
 
+def cleanup_backend_services(args):
```

Comment:
shouldn't you check if a given backend service is used by some health checks?  If it is, I think deletion will be rejected, no?

Reply:
A backendservice can be deleted even if it is used by health checks. that's why we have cleanup_health_checks for orphan health checks.